katas/content/superposition/all_basis_vectors/index.md

```diff
@@ -0,0 +1,5 @@
+**Input:** $N$ ($N \ge 1$) qubits in the $|0 \dots 0\rangle$ state (stored in an array of length $N$).
```

Comment:
I think this task description was swapped with GHZ one, since they're otherwise exactly what they should be